common/src/main/java/com/microsoft/identity/common/internal/controllers/CommandDispatcher.java

```diff
@@ -460,7 +462,8 @@ private static void commandCallbackOnTaskCompleted(@SuppressWarnings("rawtypes")
     private static void cacheCommandResult(@SuppressWarnings(WarningType.rawtype_warning) BaseCommand command,
                                            CommandResult commandResult) {
         if (command.isEligibleForCaching() && eligibleToCache(commandResult)) {
-            sCommandResultCache.put(command, commandResult);
+            // Disabling throttling ADO:1383033
+            // sCommandResultCache.put(command, commandResult);
```

Comment:
Instead of commenting these out, you can also choose to comment line 255 isntead

Reply:
whichever